src/map/script.cpp

```diff
@@ -19497,11 +19505,6 @@ BUILDIN_FUNC(unitstopwalk)
 		if (ud != nullptr)
 			ud->state.force_walk = false;
 
-		if (unit_stop_walking(bl, flag) == 0 && flag != USW_FORCE_STOP) {
-			ShowWarning("buildin_unitstopwalk: Unable to find unit or unit is not walking.\n");
-			return SCRIPT_CMD_FAILURE;
-		}
-
```

Comment:
Not really sure why you remove this?
The rest looks good!

Reply:
yeah `unit_stop_walking` should not be removed